### DISCUSSION TITLE
Incorrect Cover Art

### DISCUSSION THREAD

**Discussion by c0utta** [navidrome/navidrome#3396] [Q&A]
### I confirm that:

- [X] I have searched the existing [open AND closed issues](https://github.com/navidrome/navidrome/issues?q=is%3Aissue) to see if an issue already exists for the bug I've encountered
- [X] I'm using the latest version (your issue may have been fixed already)

### Version

0.53.3 (13af8ed4)

### Current Behavior

Setting in navidrome.toml:
CoverArtPriority="embedded, cover.*"

If a track does not have embedded art, Navidrome selects the art from the next track that has imbedded art.

### Expected Behavior

If a track does not have embedded art, Navidrome should select cover.* 

### Steps To Reproduce

1. Track 1 does not have embedded art
2. Track 2 has embedded art
3. When Track 1 plays, embedded art from Track 2 is displayed

### Environment

```markdown
- OS: Debian 11
- Browser: Edge
- Client: Browser
```

### How Navidrome is installed?

Binary (from downloads page)

### Configuration

```toml
CoverArtPriority="embedded, cover.*"
```

### Code of Conduct

- [X] I agree to follow Navidrome's Code of Conduct

**Comment by deluan** [comment]
Assuming these tracks are from the same album, and assuming the cover in Track 2 is the same you see in the Album page, this is expected behaviour: Navidrome selects the first embedded cover it finds in any track from an album as the album cover. And when showing covers for tracks, if the cover does not have any embedded art, it displays the album cover.

**Comment by c0utta** [comment]
Hi deluan,

I understand the logic now, and in conjunction with the MediaFiles section (when was that added?) of https://www.navidrome.org/docs/usage/artwork/ I have made one change.

CoverArtPriority="cover.*".

I have now achieved my goal thanks to your explanation.
